q2_types/_feature_data.py

```diff
@@ -119,24 +135,16 @@ def pandas_to_taxonomy(view, data_dir):
     _write_taxonomy(view, data_dir)
 
 
-def sequences_to_pandas_series(data_dir):
-    index = []
-    seqs = []
-    for seq in skbio.io.read(os.path.join(data_dir, 'sequences.fasta'),
-                             format='fasta'):
-        index.append(seq.metadata['id'])
-        seqs.append(seq)
-    return pd.Series(data=seqs, index=index)
-
+# TODO can this be generalized to any iterable type?
```

Comment:
one day... probably mention this: https://github.com/biocore/scikit-bio/issues/1031#issuecomment-225252290
in the TODO for the next guy